monai/transforms/utility/array.py

```diff
@@ -283,8 +285,13 @@ def __call__(
 
         if self.data_shape if data_shape is None else data_shape:
             lines.append(f"Shape: {img.shape}")
-        if self.intensity_range if intensity_range is None else intensity_range:
-            lines.append(f"Intensity range: ({np.min(img)}, {np.max(img)})")
+        if self.value_range if value_range is None else value_range:
+            if isinstance(img, np.ndarray):
+                lines.append(f"Value range: ({np.min(img)}, {np.max(img)})")
+            elif torch.is_tensor(img):
+                lines.append(f"Value range: ({torch.min(img)}, {torch.max(img)})")
+            else:
+                raise RuntimeError(f"unsupported input data type: {type(img)}.")
```

Comment:
I don't think this should be a runtime error, could print "not a pytorch/numpy array"

Reply:
updated in the latest commit.
Thanks.